Centaurus.Test.Utils/GlobalInitHelper.cs

```diff
@@ -40,34 +40,38 @@ public static AuditorSettings GetAuditorSettings()
             return settings;
         }
 
-        private static void SetCommonSettings(BaseSettings settings, string secret)
+        public static void SetCommonSettings(BaseSettings settings, string secret)
```

Comment:
I don't like those defaults. If people set `HorizonUrl` and forget `NetworkPassphrase`, things will end up badly. Can we set theses settings as mandatory? Or use some kind of presets, so that when you choose "testnet", it correctly fills up all fields? Also, `AppData` won't work on Unix (at least not in the way it's intended).

Reply:
This code is used only for tests, so I didn't spent much time for it. If you think that this is critical thing, I will fix it. 
`AppData` is just a folder where application files (mongodb files, logs) will be stored